pkg/apis/componentconfig/types.go

```diff
@@ -31,6 +31,9 @@ type KubeProxyConfiguration struct {
 	HealthzPort int `json:"healthzPort"`
 	// hostnameOverride, if non-empty, will be used as the identity instead of the actual hostname.
 	HostnameOverride string `json:"hostnameOverride"`
+	// iptablesMasqueradeBit is the bit of the iptables fwmark space to use for SNAT if using
+	// the pure iptables proxy mode. Values must be within the range [0, 31].
+	IPTablesMasqueradeBit *int `json:"iptablesMasqueradeBit"`
```

Comment:
I think you can make the internal (non-versioned) type use an int, and the versioned type use a pointer, and that way the code that actually uses the internal type doesn't have to much with dereferencing and defaulting.  The smell here is that the value `14` turns up outside of the defaults.go file at all.